### ISSUE TITLE
[Request] Allow server side GCode rendering

### ISSUE THREAD

**Issue by chatelao** [foosel/OctoPrint#3404]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when loading the GCode viewer crashes on iOS (iPad) or takes quite long.

**Describe the solution you'd like**
To render the GCode-Layers to images on the server-side and just display the minimum in the browser could help.

**Describe alternatives you've considered**
I have no clue yet, how to run OctoPi on my iPad (a nice Display option for the printer)

**Additional context**
Basically I would like to my iPad as Printer Monitor display
